pokedex/BlueButtons.js

```diff
@@ -0,0 +1,15 @@
+import Component from '../Component.js';
+
+class BlueButtons extends Component {
+
+    renderHTML() {
+        return /*html*/`
+            <div>
+                <section class="blue"><button></button><button></button><button></button><button></button><button></button><br><button></button><button></button><button></button><button></button><button></button>
```

Comment:
if these buttons are only aesthetic, it's a better practice to have them rendered via css